### DISCUSSION TITLE
Request to update Canada and US Sports, Common N/A network and News Channels

### DISCUSSION THREAD

**Discussion by Zenithity** [iptv-org/community#1101] [Q&A]
Not sure if this is an appropriate way to request this, but I noticed that there aren't many Sports (NBA League and Sportsnet) channels, Common TV stations (Discovery, History, A&E etc) and local news channels (CA: CP24 Toronto) currently in the playlist.

This may be due to lack of links which is why I decided to ask here in discussions where hopefully another user may be able to contribute.

**Comment by BellezaEmporium** [comment]
Most of them either have a stream protection (tokens, DRM), or are DMCAd.

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
